ui/raidboss/data/07-dt/raid/r11n.ts

```diff
@@ -0,0 +1,322 @@
+import { RaidbossData } from '../../../../../types/data';
+import { TriggerSet } from '../../../../../types/trigger';
+
+export interface Data extends RaidbossData {
+  weaponModels: { [string: string]: 'axe' | 'scythe' | 'sword' | 'unknown' };
+  weaponTethers: { [string: string]: string };
+  trophyActive: boolean;
+}
+
+const weaponModelIDMap: { [string: string]: 'axe' | 'scythe' | 'sword' | 'unknown' } = {
+  '11D1': 'sword',
+  '11D2': 'axe',
+  '11D3': 'scythe',
+} as const;
+
+const headMarkerData = {
+  'rawSteelSpread': '0137',
+  'massiveMeteor': '013E',
+  'greatWallOfFire': '0256',
+  'rawSteelBuster': '0258',
+  'voidStardust': '0276',
+} as const;
+
+const tetherData = {
+  'assaultTether': '00F9',
+  'foregoneTether': '0164',
+} as const;
+
+const triggerSet: TriggerSet<Data> = {
+  id: 'AacHeavyweightM3',
+  zoneId: ZoneId.AacHeavyweightM3,
+  timelineFile: 'r11n.txt',
+  initData: () => ({
+    weaponModels: {},
+    weaponTethers: {},
+    trophyActive: false,
+  }),
+  triggers: [
+    {
+      id: 'R11N Ultimate Trophy Weapons Phase',
+      type: 'StartsUsing',
+      netRegex: { id: 'B7EB', source: 'The Tyrant', capture: false },
+      run: (data) => data.trophyActive = true,
+    },
+    {
+      id: 'R11N Crown Of Arcadia',
+      type: 'StartsUsing',
+      netRegex: { id: 'B3B6', source: 'The Tyrant', capture: false },
+      response: Responses.aoe(),
+    },
+    {
+      id: 'R11N Smashdown Axe',
+      type: 'StartsUsing',
+      netRegex: { id: 'B3BA', source: 'The Tyrant', capture: false },
+      response: Responses.outOfMelee(),
+    },
+    {
+      id: 'R11N Smashdown Scythe',
+      type: 'StartsUsing',
+      netRegex: { id: 'B3BC', source: 'The Tyrant', capture: false },
+      response: Responses.getIn(),
+    },
+    {
+      id: 'R11N Smashdown Sword',
+      type: 'StartsUsing',
+      netRegex: { id: 'B3BE', source: 'The Tyrant', capture: false },
+      response: Responses.getIntercards(),
+    },
+    {
+      id: 'R11N Void Stardust',
+      type: 'HeadMarker',
+      netRegex: { id: headMarkerData['voidStardust'], capture: true },
+      condition: Conditions.targetIsYou(),
+      infoText: (_data, _matches, output) => output.spreadPuddles!(),
+      outputStrings: {
+        spreadPuddles: {
+          en: 'Spread => Bait 3x Puddles',
+        },
+      },
+    },
+    {
+      // Ensure we have clean data before each round of multi-weapon mechanics.
+      // B3CC: Trophy Weapons
+      // B7EB: Ultimate Trophy Weapons
+      id: 'R11N Trophy Weapons Initialize',
+      type: 'StartsUsing',
+      netRegex: { id: ['B3CC', 'B7EB'], source: 'The Tyrant', capture: false },
+      run: (data) => {
+        data.weaponModels = {};
+        data.weaponTethers = {};
+      },
+    },
+    {
+      id: 'R11N Assault Evolved Weapon Model Collect',
+      type: 'ActorControlExtra',
+      netRegex: { category: '0197', param1: ['11D1', '11D2', '11D3'], capture: true },
```

Comment:
I have been trying to include the AC category as a comment when using `ActorControl*Extra` triggers, as the category IDs have been changed in the past.

`0197` = `PlayActionTimeline`.